spec/features/proxy_deposit_spec.rb

```diff
@@ -36,7 +38,8 @@
       choose('Visible to the world.')
       check("I have read and accept the distribution license agreement")
       click_button("Create Article")
-      expect(page).to have_selector('#article_owner', text: user.name)
+      visit new_curation_concern_generic_work_path
```

Comment:
I'm not seeing what this line and the next are intending to do - it looks like we've just created a new Article, but we're going to create a new generic work before setting any spec expectations.

Same goes for for line 60 and 61 below.